plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php

```diff
@@ -12,23 +12,23 @@
 /**
  * Tests that product reviews page handler.
  *
- * @covers \Automattic\WooCommerce\Internal\Admin\ReviewsListTable
+ * @covers \Automattic\WooCommerce\Internal\Admin\ProductReviews\ReviewsListTable
  */
 class ReviewsListTableTest extends WC_Unit_Test_Case {
 
 	/**
-	 * Returns a new instance of the {@see ReviewsListTable} class.
+	 * Returns a new instance of the {@see \Automattic\WooCommerce\Internal\Admin\ProductReviews\ReviewsListTable} class.
 	 *
 	 * @return ReviewsListTable
 	 */
-	private function get_reviews_list_table() : ReviewsListTable {
+	private function get_reviews_list_table() : \Automattic\WooCommerce\Internal\Admin\ProductReviews\ReviewsListTable {
```

Comment:
I've seen a few class full names in this test class and in others too. Any specific reason for not importing that class and using its name only?

Reply:
Not really, just how my IDE did it when I refactored the namespace. I'll tweak it.

Reply:
Fixed in [4278670](https://github.com/godaddy-wordpress/woocommerce/pull/51/commits/42786707275a9736499c9d1a904ca65caeb9e98a).